packages/bridge-controller/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@metamask/bridge-controller",
+  "version": "1.0.0",
```

Comment:
This should not be set to 1.0.0 yet or else we run the risk of accidentally publishing this package:

``` suggestion
  "version": "0.0.0",
```